src/ui/canvas/gdi/Bitmap.cpp

```diff
@@ -45,10 +51,7 @@ Bitmap::Reset()
   if (bitmap != nullptr) {
     assert(IsScreenInitialized());
 
-#ifndef NDEBUG
-    bool success =
-#endif
-      ::DeleteObject(bitmap);
+    bool success = ::DeleteObject(bitmap);
```

Comment:
Why?

Reply:
With a defined NDEBUG you have an assert test with an undefined variable in the next line:
assert(success)
This makes no sense for me - and the macro switch has to be removed - or the assert have to chambered with  the same definition!

Reply:
With `NDEBUG`, all `assert()` calls are removed.

Reply:
Ok, I found this in the assert.h yet. 
So 'success' is an unused variable without NDEBUG ;-( 
Without this backround knowledge it is diffecult to read (and understand).
As I said: I'm very surprised how much new things I have to learn, before I can follow your style and intentions completely.

Reply:
`assert()` exists since at least C89, i.e. 32 years ago.